src/main/resources/application.properties

```diff
@@ -37,7 +37,7 @@ lms.user.read.path=/private/user/v1/read/
 lms.user.update.path=/private/user/v1/update
 progress.api.endpoint=v1/content/state/read
 participants.api.endpoint=v1/batch/participants/list
-sb.api.key=apiKey
+sb.api.key=bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJSSXZtaHRpTkxRT1lKT3dYR2xnRElReFp4bHdyZmVTZCJ9.onjwk3QTql0oZYvM-xOPuCDcBJKGTVa65J64j2hy8H0
```

Comment:
These keys are passwords.. do not commit these kind of information and make it public..
delete this branch and create new one to raise a PR